apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/CareerTimelineAndRecruitmentPage.tsx

```diff
@@ -31,7 +31,7 @@ const CareerTimelineAndRecruitmentPage = () => {
   const { userAuthInfo } = useAuthorization();
   const [{ data, fetching, error }] = useQuery({
     query: CareerTimelineExperiences_Query,
-    variables: { id: userAuthInfo?.id || "" },
+    variables: { id: userAuthInfo?.id ?? "" },
```

Comment:
Pre-existing - This mutation will fail with an emptystring user ID anway, right?